zen-librtmp.sh

```diff
@@ -27,7 +27,6 @@ function build_librtmp {
 
   pushDir rtmpdump/librtmp
 
-  sed -i "s/^SYS=posix/SYS=mingw/" Makefile
```

Comment:
The previous line was not OS-independend, but removing it breaks the mingw build (under Msys2 at least).